io.github.mujx.Nheko.json

```diff
@@ -0,0 +1,210 @@
+{
+  "id": "io.github.mujx.Nheko",
+  "command": "nheko",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.gcc8"
+  ],
+  "writable-sdk": true,
+  "build-options": {
+    "env": {
+      "CC": "/usr/lib/sdk/gcc8/bin/gcc",
+      "CXX": "/usr/lib/sdk/gcc8/bin/g++",
+      "LD_LIBRARY_PATH": "/usr/lib/sdk/gcc8/lib",
+      "V": "1"
+    }
+  },
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "build-commands": [
+        "make install"
+      ],
+      "buildsystem": "simple",
+      "name": "lmdb",
+      "sources": [
+        {
+          "sha256": "f3927859882eb608868c8c31586bb7eb84562a40a6bf5cc3e13b6b564641ea28",
+          "type": "archive",
+          "url": "https://github.com/LMDB/lmdb/archive/LMDB_0.9.22.tar.gz"
+        }
+      ],
+      "subdir": "libraries/liblmdb"
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DSPDLOG_BUILD_EXAMPLES=0 -DSPDLOG_BUILD_TESTING=0",
```

Comment:
Same, there is `cmake` buildsystem.

Reply:
Are there any docs on how to pass the custom cmake flags and what are the defaults? I tried `build-args` but it doesn't work.

Reply:
`"config-opts": ["-DFOO=BAR"]`, see `man flatpak-manifest`